.eslintrc.js

```diff
@@ -76,6 +76,7 @@ module.exports = {
       files: ['*.test.ts'],
       rules: {
         '@typescript-eslint/no-unused-expressions': 'off',
+        '@typescript-eslint/no-floating-promises': 'off',
```

Comment:
I'd be careful before merging this one, as this is one of the things that can cause flakiness. I remember dealing with problems around this in the past; I thought we had addressed all occurrences, but I presume there have been more since you created this PR. The alternative would be to go over and fix all tests with floating promises.

> cc @Torres-ssf

Reply:
It's actually quite flaky and seems to occur for all tests that include a promise. Will close for now.

Reply:
By flaky, do you mean that sometimes the lint accuses the error and sometimes it doesn't?

Reply:
Also, I believe https://github.com/FuelLabs/fuels-ts/pull/3046 should bring more reliability to this.